in-work/lln_clt.md

```diff
@@ -301,11 +458,14 @@ Y = np.sqrt(n) * (sample_means - μ)
 
 # Plot
 fig, ax = plt.subplots(figsize=(10, 6))
-xmin, xmax = -3 * s, 3 * s
+xmin, xmax = -3 * σ, 3 * σ
 ax.set_xlim(xmin, xmax)
 ax.hist(Y, bins=60, alpha=0.4, density=True)
 xgrid = np.linspace(xmin, xmax, 200)
-ax.plot(xgrid, st.norm.pdf(xgrid, scale=s), 'k-', lw=2, label='$N(0, \sigma^2)$')
+ax.plot(xgrid, st.norm.pdf(xgrid, scale=σ), 'k-', lw=2, label='$N(0, \sigma^2)$')
+ax.set_xlabel(r"$Y$", size=12)
+ax.set_ylabel(r"$density$", size=12)
```

Comment:
Maybe we should change ``density`` to ``frequency`` here?

Because the y-axis of the plot shows the frequency of outcomes, instead of density.

Also do you think it is a good idea to add a caption to the plots?

Reply:
Many thanks @shlff, I think this plot is in density, with `density=True`. The large values are because the bar width is small :)

Reply:
I also had a try on captions. I think for this plot, given I do not have contextual information/data sources, it may be better to put explanations in the main text, which follows from the previous discussions.